lms/djangoapps/instructor/tests/test_services.py

```diff
@@ -88,7 +89,8 @@ def test_reset_bad_content_id(self):
         result = self.service.delete_student_attempt(
             self.student.username,
             unicode(self.course.id),
-            'foo/bar/baz'
+            'foo/bar/baz',
+            requesting_user=self.student.username,
```

Comment:
As far as using a different user, this is the same idea as the ora2 tests - the test is checking if a user can delete themselves. So it can be a different user, it just isn't in this case.

It should definitely not be a username, the reason that wasn't failing is that these particular examples are negative tests for failures in the service before even getting to the `reset_student_attempts` call that uses this parameter. I believe they're just holdover code from the development process that I missed updating due to the not-failing. I'll clean them up.